packages/feedback/src/widget/Actor.ts

```diff
@@ -0,0 +1,59 @@
+import type { FeedbackComponent, FeedbackConfigurationWithDefaults, FeedbackTheme } from '../types';
+import { Icon } from './Icon';
+import { createElement as h } from './util/createElement';
+
+interface Props {
+  options: FeedbackConfigurationWithDefaults;
+  theme: FeedbackTheme;
+  onClick?: (e: MouseEvent) => void;
+}
+
+interface ActorComponent extends FeedbackComponent<HTMLButtonElement> {
+  /**
+   * Shows the actor element
+   */
+  show: () => void;
+  /**
+   * Hides the actor element
+   */
+  hide: () => void;
+}
+
+/**
+ *
+ */
+export function Actor({ options, theme, onClick }: Props): ActorComponent {
+  function _handleClick(e: MouseEvent) {
+    if (typeof onClick === 'function') {
```

Comment:
Do we need the extra typecheck on top of typescript? or would `if (onClick)` be enough? I'm thinking about compression, this looks like codepath that doesn't run often.